.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
```

Comment:
this is all autoformat from vscode... if an issue i can revert; but seems better to have the standard formatting